setup.py

```diff
@@ -51,6 +51,9 @@
         "update_msc": setupbase.UpdateCodiconCommand,
     },
     entry_points={
-        "console_scripts": ["qta-browser=qtawesome.icon_browser:run"],
+        "console_scripts": [
+            "qta-browser=qtawesome.icon_browser:run",
+            "qta-fonts=qtawesome:install_bundled_fonts_system_wide",
```

Comment:
I think this script should be named `qta-install-fonts-system-wide` (or something similar) to be more specific about what it does.

I know the name is quite long (so, I'm open to suggestions), but the command not only installs our fonts, it does it for the system, not for the current user.

Reply:
Maybe something like `qta-install-fonts-allusers` or `qta-install-fonts-system`? For the moment, will change the name to the one you suggested but let me know if any other alternative is better

Reply:
> Maybe something like qta-install-fonts-allusers

This sounds good to me, but adding a dash between `all` and `users`:

    qta-install-fonts-all-users